Cython/TestUtils.py

```diff
@@ -1,18 +1,19 @@
 from __future__ import absolute_import
 
 import os
-import unittest
```

Comment:
I consider reordering imports as part of a PR not great but borderline ok-ish when new imports are being added.

I do not consider reformatting arbitrary unrelated code ok. You can make `git blame` ignore dedicated formatting commits, but you can't ignore parts of a squashed commit (and this PR will be squashed on merge). Please undo the code reformatting.

Reply:
resolved [here](https://github.com/cython/cython/pull/4241/commits/7f7cda56d79cd6d72e0cfd89492c106da686cb05)